compliance_checker/suite.py

```diff
@@ -186,6 +186,7 @@ def _get_valid_checkers(self, ds, checker_names):
         the user selected names.
         """
 
+        import pdb; pdb.set_trace()
```

Comment:
Please remove.  I also sometimes use a pre-commit Git hook to ensure debugger statements aren't present.

Reply:
I don't see this in my code -- is this from the latest commit?

Reply:
Guess this is outdated, so ignore.